lib/site_prism/dsl.rb

```diff
@@ -145,13 +144,13 @@ def build(type, name, *find_args)
         if find_args.empty?
           create_error_method(name)
         else
-          map_item!(type, name)
+          map_item(type, name)
           yield
         end
         add_helper_methods(name, *find_args)
       end
 
-      def map_item!(type, name)
+      def map_item(type, name)
```

Comment:
I know it's not related to this PR, but why mapped_items is an array of hashes `type -> name`. Any use case?
IMO, it would be better to have mapped_items to be something like, `element: [:name1, :name2, :name3], section: [:section1, section2, section3]`

Reply:
This was something I changed a bit ago, and it's going to help the framework of recursion for all_there.

It could be in the form you've suggested sure. If you wanna raise a PR go for it.

Currently these are only used for `all_there?` and `mapped_items` checks